jsonschema-hyperschema.xml

```diff
@@ -390,8 +389,13 @@
 
             <section title="Links and data">
                 <t>
-                    "Form"-like functionality can be defined by use of the <xref target="method">"method"</xref> and <xref target="submissionSchema">"submissionSchema"</xref> keywords, which supplies a schema describing the data to supply to the server.
-                    Functionality equivalent to dynamic URI generation is available through the <xref target="href">"href"</xref> template and <xref target="hrefSchema">"hrefSchema"</xref>.
+                    "Form"-like functionality can be defined by use of the
```

Comment:
@jdesrosiers I realize now that I didn't fix this because the whole section gets totally rewritten in #293.  So unless you think #293 is going to get substantially held up, I'm going to leave this as-is in this PR rather than figure out better wording that will get deleted immediately.  I can always put up a small PR to address just this if #293 gets stuck anyway.